elements/lisk-transaction-pool/src/queue_checkers.ts

```diff
@@ -51,16 +66,16 @@ export const checkTransactionForId = (
 	return checkTransactionPropertyForValues(ids, idProperty);
 };
 
-export const checkTransactionForSenderIdWithRecipientIds = (
+export const checkTransactionForSenderPublicKeyWithRecipientIds = (
 	transactions: ReadonlyArray<Transaction>,
 ): ((transaction: Transaction) => boolean) => {
 	const recipientProperty: TransactionFilterableKeys = 'recipientId';
-	const senderId: TransactionFilterableKeys = 'senderId';
-	const recipients = transactions.map(
-		transaction => transaction[recipientProperty],
-	);
+	const senderIdProperty: TransactionFilterableKeys = 'senderId';
```

Comment:
This can also be reverted

Reply:
After offline discussion no change here.